File: A. Make A Equal to B/solve.cpp
#include <iostream>
using namespace std;
int main()
{
    int t, n;
    cin >> t;
    while (t--)
    {
        cin >> n;
        int i, a[n], b[n], c = 0, ca = 0, cb = 0;

        for (i = 0; i < n; i++)
        {
            cin >> a[i];
            if (a[i] == 0)
                ca++;
        }

        for (i = 0; i < n; i++)
        {
            cin >> b[i];
            if (b[i] == 0)
                cb++;
        }

        for (i = 0; i < n; i++)
            if (a[i] != b[i])
                c++;

        int x = abs(ca - cb);
        cout << min(c, x + 1) << endl;
    }
    return 0;
}

File: C. Most Similar Words/solve.py
letters = "abcdefghijklmnopqrstuvwxyz"
m = {}
for i in range(1,27):
	m[letters[i-1]] = i


def cost(s1,s2):
	t = 0
	for i,j in zip(s1,s2):
		t+=abs(m[i]-m[j])
	return t


for _ in range(int(input())):
	n,sl = map(int, input().split())
	s = []
	for i in range(n):
		s.append(input())


	min_cost = 1000
	for n,i in enumerate(s):
		for j in s[n+1:]:
			# print(i,j,cost(i,j))
			min_cost = min(min_cost,cost(i,j))

	print(min_cost)

